### ISSUE TITLE
Combined lab test result

### ISSUE THREAD

**Issue by rtdany10** [earthians/marley#3]
Lab tests are created individually. Their results are individually generated on different documents.
In a scenario where one patient does 5 tests, 5 printouts are required.

Either we could club multiple tests of same department into 1 Lab Test document or maybe keep a print format that fetches test results of that patient all in one print.

Although, I prefer to allow multiple tests in one document. In that way, it is easier to manage.

**Comment by pjamsheer** [comment]
@rtdany10 If I am not wrong please check https://docs.erpnext.com/docs/v13/user/manual/en/healthcare/lab_test_template#14-grouped-result-format and you can create print format from invoice to print all the labtest in one print !!

**Comment by rtdany10** [comment]
> @rtdany10 If I am not wrong please check https://docs.erpnext.com/docs/v13/user/manual/en/healthcare/lab_test_template#14-grouped-result-format and you can create print format from invoice to print all the labtest in one print !!

Hai @pjamsheer, assume I have 100 different lab tests. If I am to group lab tests, the total number of lab test templates are going to be huge.

Plus, doctors are not going to select grouped lab test template in encounter. They'll select individual tests by name as each patient require different tests.

Grouping lab test has another purpose but unfortunately can't be used for this one.

**Comment by akurungadam** [comment]
> keep a print format that fetches test results of that patient all in one print.

Yes, we should have a print format which would print all lab tests for the Patient, Lab Tests based on Invoice or Encounter similar to 'Create multiple" option available in Lab Test list view.

**Comment by aakvatech** [comment]
https://github.com/aakvatech/hms_tz/tree/main/hms_tz/hms_tz/print_format/lab_test_results

You may modify and use this. This is for all lab test related to that patient ordered in the same group of encounters. It's specific to our requirements, but it may be useful for you.

![image](https://user-images.githubusercontent.com/35020381/133936784-2f88e963-1cb9-4734-b266-b981d7cf3fab.png)

**Comment by mujeerhashmi** [comment]
> Yes, we should have a print format which would print all lab tests for the Patient, Lab Tests based on Invoice or Encounter similar to 'Create multiple" option available in Lab Test list view. 

We have seen the most effective & efficient way is to let the user select lab tests from the list-view and print a merged report of selected lab tests.

**Comment by mujeerhashmi** [comment]
I have raised this [PR](https://github.com/frappe/erpnext/pull/27642).

**Comment by akurungadam** [comment]
Diagnostic Report (version-15 onwards) groups Observations based on Sales Invoice / Order Source (Patient Encounter)
